certbot/certbot/_internal/main.py

```diff
@@ -113,12 +113,20 @@ def _get_and_save_cert(le_client, config, domains=None, certname=None, lineage=N
         if lineage is not None:
             # Renewal, where we already know the specific lineage we're
             # interested in
-            logger.info("Renewing an existing certificate")
+            logger.info(
+                "Renewing%s an existing certificate for %s",
+                " (dry run)" if config.dry_run else "",
+                display_util.summarize_domain_list(domains or lineage.names()),
+            )
             renewal.renew_cert(config, domains, le_client, lineage)
         else:
             # TREAT AS NEW REQUEST
             assert domains is not None
-            logger.info("Obtaining a new certificate")
+            logger.info(
+                "Requesting%s a certificate for %s",
```

Comment:
What if we do "Requesting a certificate for %s" / "Simulating a request of a certificate for %s"

Reply:
I had tried it, but:

1. The simulating version is a bit wordy - I found myself having to manually parse it in my head.
2. Is "Simulating" a sufficiently strong member of the Certbot vocabulary to stand on its own? Wherever it currently appears, it appears next to the words "dry run". I am worried that if we don't include the words "dry run", the mental connection between the user input of `--dry-run` and the printed word `Simulating` won't be made.
3. `(dry run simulation)` is another variant I had tried, but didn't like it either.

Reply:
1. Good point, we can do better
2. I'm not too worried about not including "dry run" necessarily, since the user presumably knows that's what happening since they put "dry run" in the command. But I can see your point on simulating possibly not being strong enough.

I think my actual objection here is about the clunkiness of "Requesting (dry run)", since I'd expect something in parentheses to parse grammatically. What do you think about just switching it around to remove that expectation, like "[dry run] Requesting..."?

Reply:
OK. Maybe then a slight tweak to the original proposition ("a request of a certificate" → "a certificate request"):

-  Requesting a certificate for %s / Simulating a certificate request for %s
-  Renewing an existing certificate for %s / Simulating renewal of an existing certificate for %s

I did notice though that because it's planned to drop the following ...

https://github.com/certbot/certbot/blob/6c7b99f7e0faa60b595b06933f292d476b85799c/certbot/certbot/_internal/renewal.py#L432

... it's no longer clear what lineage is being renewed.

What do you think? 

It could say "Renewing certificate {cert_name}" instead of printing the SAN list. The certificate name is probably the more relevant information when it comes to renewal. 

>Simulating renewal of certificate example.comz
>Renewing certificate example.org

Or even both:

>Simulating renewal of /etc/letsencrypt/renewal/example.comz.conf with domains: example.comz and 2 others
>Renewing /etc/letsencrypt/renewal/example.org.conf with domains: example.org

Reply:
> Simulating renewal of /etc/letsencrypt/renewal/example.comz.conf with domains: example.comz and 2 others
> Renewing /etc/letsencrypt/renewal/example.org.conf with domains: example.org

Let's go with this and if we happen to come up with something better later we can always change it